assignments/closure.js

```diff
@@ -1,6 +1,19 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+const author = "jrrTolkien";
+const middleEarth = function() {
+  const elves = function() {
+    const legolas = function() {
+
+    }
+  }
+}
+
```

Comment:
```suggestion

const name = "Clark"

function closureExample() {
  return name
}

console.log(closureExample())
```

Reply:
remember: this is an example of a closure. The function returns the value of name by looking inside the function where the function has to get the value of name from the global scope.